app/controllers/users_controller.rb

```diff
@@ -19,9 +28,61 @@ def create
     end
   end
 
+
+    if @user.edit user_params
```

Comment:
missing def edit and def update

Reply:
done

Reply:
remove all code in action edit because it wrong 
jusst only need
def edit; end

Reply:
Done